middlewares/validators/search.validator.js

```diff
@@ -11,4 +11,24 @@ module.exports = {
         VALIDATOR.booleanValidator("query", "expand", true),
         VALIDATOR.searchValidator("query", "q")
     ],
+    statusValidator: [
```

Comment:
Does this validator validate for status? It seems not necessarily a search and update status validator, but rather a search and update hacker validator?

Reply:
Also, note that you can chain validator middlewares in the route as well! Consider that both your new validators perform the function of 'check for valid search' + 'check for something else', and searchQueryValidator already performs the first part of that. Instead of having your validators duplicate this code, you could chain searchQueryValidator as well as statusValidator in the route.